application/modules/payments/views/online_logs.php

```diff
@@ -0,0 +1,63 @@
+<div id="headerbar">
```

Comment:
Where does that file come from? Seems to be incorrect, git shows conflicts wit this file.

Reply:
The last version 1.5.9 exists this file, I just copied it and added the paging line.